core/sis-utility/src/main/java/org/apache/sis/math/Statistics.java

```diff
@@ -310,7 +310,6 @@ public void accept(final long sample) {
     public void combine(final Statistics stats) {
         ArgumentChecks.ensureNonNull("stats", stats);
 
-        // "if (a < b)" is equivalent to "if (!isNaN(a) && a < b)".
```

Comment:
Why removing the comment?

Reply:
It seemed unnecessary.
Therefore I removed it to increase readability.

Going to undo the change.

Reply:
The key element in the comment was the `isNaN(a)` part. Not so many developers are aware that `(a < b)` and `!(a >= b)` are *not* equivalent when using floating point numbers.